airbyte-ci/.gitignore

```diff
@@ -30,7 +30,6 @@ MANIFEST
 #  Usually these files are written by a python script from a template
 #  before PyInstaller builds the exe, so as to inject date/other infos into it.
 *.manifest
-*.spec
```

Comment:
Could you share why it's not gitignored anymore?

Reply:
Ah, this was an artifact from attempting to use spec files as the way to define pyinstaller configuration.

will revert